### ISSUE TITLE
Fluid pipe render issues

### ISSUE THREAD

**Issue by LuigiHutch** [AlexIIL/Buildcraft_1.8_Issues#61]
Buildcraft 7.2.0 Prerelease 1

Fluid Pipes dont render fluids properly see pics:
![2015-12-17_22 26 51](https://cloud.githubusercontent.com/assets/9326429/11883853/91722588-a50d-11e5-911a-2d0bed271b12.png)

 -LuigiHutch

**Comment by Xariyl** [comment]
It appears that fluids are rendering OUTSIDE the pipe when the pipes are full.

![Imgur](http://i.imgur.com/hyGN0Xr.png)

**Comment by AlexIIL** [comment]
So it turns out I wasn't reading the data correctly from the network, causing it to get confused about which side had fluid

**Comment by Xariyl** [comment]
Confirm fixed as of buildcraft-7.2.0pre4.

**Comment by AlexIIL** [comment]
from #70 

```
Ah sorry for not getting back to you: I was thinking of letting you close issues after 
you can confirm that the problem you had is fixed. This was I don't prematurely 
close an issue just because I no longer noticed it, as I may have missed something :)
```
